app/src/utils/utils.ts

```diff
@@ -3,21 +3,35 @@
  */
 import router from 'src/router/index';
 import { RouteLocationRaw } from 'vue-router';
-import { BigNumber, isAddress } from 'src/utils/ethers';
-import { BigNumberish, Contract, ContractTransaction } from 'ethers';
-import { GrantRound } from '@dgrants/types';
+import { BigNumber, BigNumberish, Contract, ContractTransaction, isAddress } from 'src/utils/ethers';
+import { SUPPORTED_TOKENS_MAPPING } from 'src/utils/constants';
+import { CartItem, CartItemOptions } from 'src/types';
+import { Donation, Grant, GrantRound, SwapSummary } from '@dgrants/types';
 
+// --- Formatters ---
 // Returns an address with the following format: 0x1234...abcd
 export function formatAddress(address: string) {
   if (!address || address.length !== 42) return null;
   return `${address.slice(0, 6)}...${address.slice(38)}`;
 }
 
-// Navigates to the specified page and pushes a new entry into the history stack
-export async function pushRoute(to: RouteLocationRaw) {
-  await router.push(to);
+// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
+export function daysAgo(val = 0) {
```

Comment:
push this into utils package ?

Reply:
I think the question to answer here is "what goes into the shared utils package, and what goes into the shared types package?" I've been organizing things as if the answer is "only utils/types that _are_ shared between other packages". If we think that answer should be "any utils/types that _may_ be helpful to someone integrating their own frontend", then I agree we should move it. 

Given that, I'd say we leave this as-is and figure this out in a separate discussion, which I created here: https://github.com/dcgtc/dgrants/discussions/114

(Side note: I rearranged this file a bit which is why it looks like there's a lot of changes).

Reply:
Haha alrighty ! thanks for creating the discussion